src/data/moves/move-attrs/cut-hp-stat-stage-boost-attr.ts

```diff
@@ -41,8 +43,33 @@ export class CutHpStatStageBoostAttr extends StatStageChangeAttr {
     return ret;
   }
 
-  override getCondition(): MoveConditionFunc {
+  public override getCondition(): MoveConditionFunc {
     return (user, _target, _move) =>
       user.getHpRatio() > 1 / this.cutRatio && this.stats.some((s) => user.getStatStage(s) < 6);
   }
+
+  /**
+   * @returns An additional penalty for reducing the user's HP to go with the score
+   * for this effect's stat stage changes. This checks each of the user's opponents'
+   * max {@link Pokemon.getExpectedAttackScore | EAS} among their estimated attacks
+   * to predict how much damage the user will receive this turn. If the user is
+   * projected to receive more damage than the effect's {@linkcode cutRatio}, this
+   * grants a {@linkcode BAD_MOVE_PENALTY}.
+   */
+  public override getRawEffectScore(user: EnemyPokemon, target: Pokemon, move: Move): number {
+    const oppMaxEas = user.getOpponents().map((opp) =>
+      opp.estimateAttackMoves().reduce((maxEas, move) => {
+        const eas = opp.getExpectedAttackScore(user, move);
```

Comment:
https://biomejs.dev/linter/rules/no-shadow/
```suggestion
      opp.estimateAttackMoves().reduce((maxEas, mv) => {
        const eas = opp.getExpectedAttackScore(user, mv);
```